views/action-form-add.twig

```diff
@@ -58,19 +58,24 @@
                 {% set next %}{% trans "Next" %}{% endset %}
                 {% set previous %}{% trans "Previous" %}{% endset %}
                 {% set navLayout %}{% trans "Navigate to Layout" %}{% endset %}
-                {% set navRegion %}{% trans "Navigate to Region" %}{% endset %}
                 {% set navPlaylist %}{% trans "Navigate to Playlist" %}{% endset %}
                 {% set navWidget %}{% trans "Navigate to Widget" %}{% endset %}
                 {% set options = [
                     { typeid: "next", type: next },
                     { typeid: "previous", type: previous },
                     { typeid: "navLayout", type: navLayout },
-                    { typeid: "navRegion", type: navRegion },
-                    { typeid: "navPlaylist", type: navPlaylist },
                     { typeid: "navWidget", type: navWidget },
                 ] %}
                 {{ forms.dropdown("actionType", "single", title,"", options, "typeid","type", helpText) }}
 
+                {% set title %}{% trans "Widget ID" %}{% endset %}
```

Comment:
We have called this field Widget ID, but actually we are showing a list of Widgets (not IDs) - should we rename?